packages/xo-server-load-balancer/src/index.js

```diff
@@ -94,63 +92,26 @@ export const configurationSchema = {
 
 // ===================================================================
 
-// Create a job not enabled by default.
-// A job is a cron task, a running and enabled state.
-const makeJob = (cronPattern, fn) => {
-  const job = {
-    running: false,
-    emitter: new EventEmitter(),
-  }
-
-  job.cron = new CronJob(cronPattern, async () => {
-    if (job.running) {
-      return
-    }
-
-    job.running = true
-
-    try {
-      await fn()
-    } catch (error) {
-      console.error(
-        '[WARN] scheduled function:',
-        (error && error.stack) || error
-      )
-    } finally {
-      job.running = false
-      job.emitter.emit('finish')
-    }
-  })
-
-  job.isEnabled = () => job.cron.running
-
-  return job
-}
-
-// ===================================================================
-// ===================================================================
-
 class LoadBalancerPlugin {
   constructor (xo) {
     this.xo = xo
-    this._job = makeJob(
-      `*/${EXECUTION_DELAY} * * * *`,
-      this._executePlans.bind(this)
+
+    this._job = createSchedule(`*/${EXECUTION_DELAY} * * * *`).createJob(
```

Comment:
Default timezone is UTC, is it what we want?

Reply:
I think it will not have an impact because the job is executed every minutes.